music_assistant/server/providers/ytmusic/__init__.py

```diff
@@ -49,6 +55,7 @@
 #     from collections.abc import AsyncGenerator
 
 CONF_COOKIE = "cookie"
+LOGGER = logging.getLogger(__name__)
```

Comment:
no need for this, you have an instance variable logger
so self.logger is available